applications_tests/dem_3d/insert_plane_3d.output

```diff
@@ -0,0 +1,43 @@
+
+*********************
+Running on 1 rank(s)
+*********************
+DEM time-step is 7.30826% of Rayleigh time step
+Reading triangulation
+
+Finished reading triangulation
+Warning: expansion of particle-wall contact list is disabled.
+This feature is useful in geometries with concave boundaries.
+id, type, dp, x, y, z
+0 0 0.01000 -0.2910 -0.4639 -0.2967
+1 0 0.01000 -0.0452 -0.4649 -0.3552
+2 0 0.01000 -0.3415 -0.4624 -0.0972
+3 0 0.01000 -0.0696 -0.4630 -0.0621
+4 0 0.01000 0.1615 -0.4641 -0.2798
+5 0 0.01000 0.4666 -0.4640 -0.3033
+6 0 0.01000 0.1392 -0.4643 -0.1234
+7 0 0.01000 0.3993 -0.4635 -0.0446
+8 0 0.01000 -0.3593 -0.4646 0.1380
+9 0 0.01000 -0.1141 -0.4655 0.1468
+10 0 0.01000 -0.3237 -0.4651 0.4363
+11 0 0.01000 -0.0954 -0.4643 0.4274
+12 0 0.01000 0.1744 -0.4654 0.1543
+13 0 0.01000 0.4521 -0.4640 0.2020
+14 0 0.01000 0.1650 -0.4653 0.4033
+15 0 0.01000 0.4102 -0.4645 0.4669
+16 0 0.01000 0.3820 -0.0696 -0.0724
+17 0 0.01000 -0.3664 -0.0976 -0.3087
+18 0 0.01000 -0.2860 -0.0807 0.1314
+19 0 0.01000 -0.1230 -0.0569 -0.1187
+20 0 0.01000 0.1488 -0.0001 -0.2848
+21 0 0.01000 0.4601 -0.0680 0.1790
+22 0 0.01000 0.1625 -0.0060 -0.0737
+23 0 0.01000 0.4418 -0.0241 -0.3711
+24 0 0.01000 -0.0812 0.0159 0.4681
+25 0 0.01000 -0.3029 -0.0294 -0.0511
+26 0 0.01000 0.4390 -0.0166 0.4438
+27 0 0.01000 -0.3584 -0.0034 0.4630
+28 0 0.01000 0.2079 -0.0069 0.3979
+29 0 0.01000 -0.0357 -0.0020 -0.3063
+30 0 0.01000 0.2206 0.0303 0.1907
+31 0 0.01000 -0.0391 0.0187 0.2174
```

Comment:
I think it has to do with the generation of the random number and the seed. 

Since both core have the same seed, I think that the n_th random number generated by core 1 will be the same as the n_th random number generated by core 2.